### PR TITLE
Fix BC break

### PR DESCRIPTION
**Pull request by merk**

### CODE REVIEW

File: lib/Elastica/ResultSet.php [hunk 1]
```diff
@@ -62,6 +86,12 @@ public function __construct(Response $response, Query $query, $results)
         $this->_query = $query;
         $this->_response = $response;
         $this->_results = $results;
+
+        $data = $response->getData();
+        $this->_maxScore = isset($data['hits']['max_score']) ? (float) $data['hits']['max_score'] : 0;
```
Comment (ruflin) [inline lib/Elastica/ResultSet.php]: Could we keep this code in the getters and use here the getters instead? It means we use getData() 4 times but that should be ok?

**Comment by ruflin** [comment]
Thanks. Failing test is because of a composer problem: https://github.com/composer/composer/issues/5308

**Comment by merk** [comment]
IMO, its just neater to leave it this way until we remove the properties

File: lib/Elastica/ResultSet.php (hunk 1 above)
Comment (ruflin) [inline lib/Elastica/ResultSet.php]: Doesn't matter too much to me :-) Will merge as it is.

**Pull request merged**